### PR TITLE
Basic updates to meet design for sections

### PR DESCRIPTION
**Pull request by justtry**
Updates in this PR will be used for all section, for example https://github.com/x-team/auto/issues/678

### CODE REVIEW

File: src/components/Text.js [hunk 1]
```diff
@@ -36,7 +37,16 @@ const Heading = elem.h1(cmz(
   `
 ))
 
-const SubHeading = elem.h2(cmz(`
+const SectionHeading = elem.h2(cmz(`
+  font-weight: 800;
+  font-size: 36px;
+  padding-top: 64px 0 0 0;
+  letter-spacing: -.025em;
+  font-family: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
```
Comment (nicksp) [inline src/components/Text.js]: I did a bunch of typography changes in here https://github.com/x-team/auto-ui/pull/77 Also I did some changes to the `Text` component by accepting two new props: `subHeadingSize` and `headingSize` which covers what you're doing here.

Comment (nicksp) [reply src/components/Text.js]: So the whole PR might not be needed.

**Comment by nicksp** [comment]
Let's just keep some other things that I didn't touch in my PR.

File: src/components/Text.js (hunk 1 above)
Comment (justtry) [reply src/components/Text.js]: then we totally need to merge your PR asap, as currently most of the text style are off

Comment (nicksp) [reply src/components/Text.js]: right, I agree. @bernardodiasc is reviewing that afaik

**Comment by nicksp** [comment]
Once https://github.com/x-team/auto-ui/pull/77 is merged, will get back to this one and see if it's still relevant.

**Comment by justtry** [comment]
@nicksp I can remove fonts changes there if your PR covers everything and keep only milestones and spaces fix

**Comment by nicksp** [comment]
That would be awesome :) thanks



Sent from myMail for iOS


Wednesday, 13 December 2017, 08:13 +0300 from notifications@github.com  <notifications@github.com>:

**Comment by bernardodiasc** [comment]
Hi @justtry, about milestones spacing fix, can you take a look how this https://github.com/x-team/auto-ui/pull/76 affects (or not) what you made? In that PR contains the MilestonesScreen, that is the layout of the milestones pages content. Cheers!

**Comment by nicksp** [comment]
@justtry Please merge with #77 and reference #76 in order to make changes to this PR.

**Comment by justtry** [comment]
Closing and will just open follow up pr for milestone

**Pull request closed without merging**